tests/test_cocos.py

```diff
@@ -153,6 +153,38 @@ def test_valid_numerator_of_unit_one(self):
         model = self._parse_and_validate_model(os.path.join(os.path.realpath(os.path.join(os.path.dirname(__file__), 'valid')), 'CoCoUnitNumeratorNotOne.nestml'))
         assert len(Logger.get_all_messages_of_level_and_or_node(model, LoggingLevel.ERROR)) == 0
 
+    def test_invalid_fractional_unit_exponent(self):
+        model = self._parse_and_validate_model(os.path.join(os.path.realpath(os.path.join(os.path.dirname(__file__), 'invalid')), 'fractional_unit_exponent.nestml'))
```

Comment:
Would it be better to provide the exact number of errors here, similar to other invalid tests?

Reply:
I have instead partially implemented https://github.com/nest/nestml/issues/561 so that we can check even more accurately whether specific message codes appear in the log.